### PR TITLE
feat: [Home] Add trending artists & auction lots rail

### PR DESCRIPTION
**Pull request by damassi**
Addresses 
- https://artsyproduct.atlassian.net/browse/GRO-524
- https://artsyproduct.atlassian.net/browse/GRO-520

This adds a new Trending Artists rail to the home page: 

Screen Shot 2021-09-20 at 4 19 44 PM

As well as the Auction Lots rail: 

Screen Shot 2021-09-20 at 5 28 15 PM

### CODE REVIEW

File: src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  if (nodes.length === 0) {
+    return null
+  }
+
+  return (
+    <HomeAuctionLotsRailContainer lotCount={nodes.length}>
+      <Shelf>
+        {nodes.map((node, index) => {
+          return (
+            <Box>
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx:42]: Why the extra `Box`? Would have to move the `key` up if there's a reason

File: src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+              <ShelfArtworkFragmentContainer
+                artwork={node}
+                key={node.slug}
+                // TODO: Add home type to cohesion once we have tracking
+                contextModule={null as any}
+                hidePartnerName
+                // FIXME: For some reason this is leaves the gray loading
+                // background post load
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx:50]: That's always there (it's on my grievances list). But why is it a problem in this case? It should only matter for transparent images.

File: src/v2/Apps/Home/Components/HomeTrendingArtistsRail.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+                <Spacer mt={1} />
+
+                <EntityHeader
+                  name={node.name!}
+                  meta={node.formattedNationalityAndBirthday!}
+                  smallVariant
+                  FollowButton={
+                    <FollowArtistButtonFragmentContainer
+                      user={user}
+                      artist={node as any}
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeTrendingArtistsRail.tsx:73]: Why's this get cast as `any`?

File: src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx (hunk 2 above)
Comment (damassi) [reply src/v2/Apps/Home/Components/HomeAuctionLotsRail.tsx:50]: Wrong width set for image, fixed

**Pull request merged**